doc/docutils.rst

```diff
@@ -0,0 +1,57 @@
+Sphinx vs. docutils
+===================
+
+tl;dr: docutils converts reStructuredText to multiple output formats. Sphinx
+builds upon docutils to allow construction of cross-referenced and indexed
+bodies of documentation.
+
+docutils
+--------
+
+`docutils`__ is a text processing system for converting plain text
+documentation into other, richer formats. docutils uses *readers* to read a
+document, *parsers* for parsing plain text formats into an internal tree
+representation made up of different types of *nodes*, and *writers* to output
+this tree in various document formats [1]_.  docutils provides parsers for one
+plain text format - `reStructuredText`__ - though other, *out-of-tree* parsers
+have been implemented including Sphinx's :doc:`Markdown parser <markdown>`. On
+the other hand, it provides writers for many different formats including HTML,
+LaTeX, man pages, Open Document Format and XML [2]_.
+
+docutils exposes all of its functionality through a variety of front-end tools,
+such as `rst2html`, `rst2odt` and `rst2xml` [3]_. Crucially though, all of
+these tools, and docutils itself, are concerned with individual documents. They
+don't support concepts such as cross-referencing, indexing of documents, or
```

Comment:
What did you mean "cross-reference"? Surely docutils does not support cross-references over the file because docutils is designed to process a single file. But it provides a way to refer a hyperlink target in the same file. I believe the main difference between docutils and Sphinx is number of target files.

Reply:
```suggestion
don't support concepts such as cross-referencing between multiple documents, indexing of documents, or
```